app/controllers/idv/outage_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Idv
+  class OutageController < ApplicationController
+    include IdvSession
+    include StepIndicatorConcern
+
+    before_action :confirm_two_factor_authenticated
+
+    def show
+      session[:skip_vendor_outage] = true
+      render :show, locals: { current_sp: current_sp, exit_url: get_exit_url }
+    end
+
+    def get_exit_url
```

Comment:
Conventionally I wouldn't expect the `get_` prefix for a Ruby method like this.

```suggestion
      render :show, locals: { current_sp: current_sp, exit_url: exit_url }
    end

    def exit_url
```

If you wanted, you could also use the new Ruby 3.2 shorthand syntax:

```suggestion
      render :show, locals: { current_sp:, exit_url: }
    end

    def exit_url
```